packages/edit-site/src/components/editor/index.js

```diff
@@ -140,7 +131,9 @@ export default function EditSiteEditor( { isPostsList = false } ) {
 		'edit-site-editor__loading-progress'
 	);
 
-	const settings = useSpecificEditorSettings();
+	const settings = useSpecificEditorSettings(
+		context?.postId && context?.postType !== 'post'
```

Comment:
```suggestion
		!! context?.postId && context?.postType !== 'post'
```

Nit: Let's ensure we always pass to a boolean value.